src/sqlancer/postgres/ast/PostgresSelect.java

```diff
@@ -56,6 +56,29 @@ public PostgresDataType getExpressionType() {
         }
     }
 
+    public static class PostgresCTE implements PostgresExpression {
```

Comment:
Do you think that `PostgresSubquery` would be a better name? I noticed that you changed `CTE` to `subquery` also in other places. If yes, It would make sense to perform the naming change also in the other files.